### PR TITLE
Delete extra examples manifests

### PR DESCRIPTION
**Pull request by pintohutch**
Delete extra manifests and move example PodMonitoring in-line with app.

### CODE REVIEW

File: examples/example-app.yaml [hunk 1]
```diff
@@ -52,3 +52,17 @@ spec:
...
+apiVersion: monitoring.googleapis.com/v1
+kind: PodMonitoring
+metadata:
+  name: prom-example
+  labels:
+    app.kubernetes.io/name: prom-example
+spec:
+  selector:
+    matchLabels:
+      app: prom-example
```
Comment (TheSpiritXIII) [inline examples/example-app.yaml:65]: Should this selector be `app` or `app.kubernetes.io/name`? It does not match the actual label name. Note, the `Deployment` uses `app`.

**Comment by yqlu** [comment]
Context: https://github.com/GoogleCloudPlatform/prometheus-engine/pull/354#issuecomment-1293924524

Should we hold off for a couple of weeks? The English docs have been updated (https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/node_exporter links to examples/node_exporter/node_exporter.yaml) but I just took a quick look at other languages and the change is still propagating...

Specifically [Deutsch](https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters?hl=de) and [Portugese](https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters?hl=pt-br) still have the "old" page live that embeds these docs, when the other languages correctly redirect this old page `/docs/managed-prometheus/exporters` --> the new landing page `/docs/managed-prometheus/exporters/introduction`

**Comment by pintohutch** [comment]
Yea - there's no rush here. Happy to let it sit a few weeks.

File: examples/example-app.yaml (hunk 1 above)
Comment (pintohutch) [reply examples/example-app.yaml:65]: I went with `app.kubernetes.io/name` for everything, as that's convention in our other [examples](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/v0.6.0-rc.0/examples/aerospike/pod-monitoring.yaml#L20).

**Pull request merged**